platforms.sh

```diff
@@ -292,15 +292,15 @@ init_desktop()
 {
   local THIS="platforms.sh>env_dispatch()>init_desktop()"
   scenarioMessage "$THIS: Setting platforms-specific parameters."
-  HPCENV=jason-desktop.seahorsecoastal.com
+  HPCENV=desktop.seahorsecoastal.com
   QUEUESYS=mpiexec
   QUEUENAME=mpiexec
   SERQUEUE=cmd
   QCHECKCMD="ps -aux | grep mpiexec "
   SUBMITSTRING="mpiexec "
   SCRATCH=/srv/asgs
-  SSHKEY=id_rsa_jason-desktop
-  ADCOPTIONS='compiler=gfortran MACHINENAME=jason-desktop'
+  SSHKEY=id_rsa_desktop
+  ADCOPTIONS='compiler=gfortran MACHINENAME=desktop'
```

Comment:
where is this used and is ADCIRC via `build adcirc` not working for you here?

Reply:
Just removing my name as part of redacting personally identifiable information. These settings are vestigial and should be removed from all platform settings.